lab_1/main.py

```diff
@@ -74,10 +135,23 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
 
+    if not (isinstance(unknown_profile, dict) and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict) and isinstance(top_n, int)):
+        return None
+    first = compare_profiles(unknown_profile, profile_1, top_n)
+    second = compare_profiles(unknown_profile, profile_2, top_n)
+    if first > second:
+        language = profile_1.get('name')
+    elif first < second:
+        language = profile_2.get('name')
+    else:
+        language = sorted([profile_1.get('name'), profile_2.get('name')])[0]
```

Comment:
explain, please, this line in writing

Reply:
if both languages have the same proportion of common words with the unknown language, it sorts language names in alphabetical order and chooses the first one